curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-intermediate-algorithmic-thinking-by-building-a-dice-game/64ae5d8d2ae28b030cfd035e.md

```diff
@@ -0,0 +1,368 @@
+---
+id: 64ae5d8d2ae28b030cfd035e
+title: Step 50
+challengeType: 0
+dashedName: step-50
+---
+
+# --description--
+
+You should apply the `forEach` method to the `scoreInputs` array with a callback function with an argument named `input`.
```

Comment:
I would make the description different from the hint. Something like: "Now you need to use the `forEach` method..."